addons/point_of_sale/models/pos_session.py

```diff
@@ -92,6 +92,16 @@ class PosSession(models.Model):
     is_in_company_currency = fields.Boolean('Is Using Company Currency', compute='_compute_is_in_company_currency')
     update_stock_at_closing = fields.Boolean('Stock should be updated at closing')
     bank_payment_ids = fields.One2many('account.payment', 'pos_session_id', 'Bank Payments', help='Account payments representing aggregated and bank split payments.')
+    sequence_login_number_id = fields.Many2one(
```

Comment:
you cannot add new fields in stable versions
https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean

Reply:
@vlst-odoo 
maybe if we target master branch, would be correct solution

@emtz10 for stable branches, we could avoid the field